lib/omnibus-ctl.rb

```diff
@@ -437,8 +437,9 @@ def package_name
 
     # returns nil when chef-server-running.json does not exist
     def running_config
-      @running_config ||= if File.exist?("#{etc_path}/chef-server-running.json")
-                            JSON.parse(File.read("#{etc_path}/chef-server-running.json"))
+      fname = "#{etc_path}/#{::ChefUtils::Dist::Server::SERVER}-running.json"
```

Comment:
This is missing the require of chef utils that makes it work. Check the failing specs

Reply:
My fault, fixed @tas50.